src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs

```diff
@@ -230,6 +233,7 @@ public override void Dispose()
             hwm.EvaluationCompleted -= hwm_EvaluationCompleted;
             hwm.SetNodeDeltaState -= hwm_SetNodeDeltaState;
             RunSettingsViewModel.PropertyChanged -= RunSettingsViewModel_PropertyChanged;
+            RunSettingsViewModel.Dispose();
```

Comment:
I don't see `RunSettingsViewModel` implementing an `IDisposable` nor a `Dispose` method defined on it. Am I missing something?